src/components/common/ui/form/input-mui/Input.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from 'react';
+import {
+  FormControl,
+  FormHelperText,
+  InputLabel,
+  OutlinedInput,
+} from '@mui/material';
+import { SxProps, Theme } from '@mui/material/styles';
+import { useField } from 'formik';
+
+import {
+  getLeftIcon,
+  getRightIcon,
+  getState,
+} from '@/components/common/ui/form/input-mui/util';
+import mergeSx from '@/lib/utils/MergeSxStylesUtil';
+
+import * as styles from './Input.styles';
+
+export enum InputState {
+  DISABLED = 'disabled',
+  DEFAULT = 'default',
+  SUCCESS = 'success',
+  ERROR = 'error',
+}
+
+export enum InputSize {
+  LARGE = 'large',
+  MEDIUM = 'medium',
+}
+
+export enum InputType {
+  DEFAULT = 'text',
+  PASSWORD = 'password',
+  SEARCH = 'search',
+}
+
+interface InputProps
+  extends Omit<React.ComponentPropsWithoutRef<'input'>, 'size'> {
```

Comment:
is it supposed to be default props for HTML input element? if yes, then use `HTMLProps<HTMLInputElement>`. and u don't need `Omit` cause u can override `size` prop in interface easily